data/presets/shop/butcher.json

```diff
@@ -14,5 +14,9 @@
     "tags": {
         "shop": "butcher"
     },
-    "name": "Butcher"
+    "name": "Butcher",
+    "aliases": [
+        "Butcher Shop",
+        "Pork Store"
```

Comment:
see https://github.com/openstreetmap/id-tagging-schema/pull/475#discussion_r886136929

Reply:
By the way, (not sure if it is the case in this instance but) if it is relevant whether it is a colloquial term or not, the documentation should be amended to mention this. 

Personally, I can't think of a reason not to include colloquial names for things, as long as they don't use bad language. One colloquial alias springs to mind: "wheelbenders" for "wall loops".

Reply:
It's common in Italian American areas.

Reply:
Right. :+1: I was initially a bit hesitant because I thought this could lead to a potential large increase of aliases if we include colloquial names as well, but the truth is that we want to include those at least as `terms` anyway, so the maintenance effort doesn't really increase.

PS: from what I see now, _Pork Store_ seems to be more like a regional term which almost exclusively used in the New York area. But that's also not an argument against including such aliases, as long as they don't have a different/conflicting meaning in another region.